packages/bpk-component-pagination/src/BpkPaginationPage.js

```diff
@@ -37,7 +37,7 @@ const BpkPaginationPage = (props) => {
   return (
     <BpkButton
       primaryOnDark={!isSelected}
-      primary={isSelected}
+      primaryOnLight={isSelected}
```

Comment:
this works fine here visually as on web we only support light mode, but the primary and primaryOnLight are 2 different styles, the only difference being on dark mode
I think we can just remove this line completely as if you don't pass one of the button types as a prop it will return by default the primary button